src/stactools/core/utils/antimeridian.py

```diff
@@ -64,15 +68,18 @@ def split(polygon: Polygon) -> Optional[MultiPolygon]:
     If the polygon does not cross the antimeridian, returns None. Only handles
     exterior rings (can't handle interior).
 
-    NOTE: Will not work on polygons that enclose the north or south poles.
-    TODO: Fix this
+    Note:
+        Will not work on polygons that enclose the north or south poles.
+
+    Todo:
+        Fix this
```

Comment:
I think I suggested including these back in the day. Feel free to clear out the Todo and whatever else.